sensorhub-driver-android/src/main/java/org/sensorhub/impl/sensor/android/video/AndroidCameraOutputVP8.java

```diff
@@ -40,15 +43,25 @@ public AndroidCameraOutputVP8(AndroidSensorsDriver parentModule, int cameraId, S
     }
 
 
+    @Override
+    protected void initOutputStructure() {
+        // create SWE Common data structure and encoding
+        VideoCamHelper fac = new VideoCamHelper();
+        // use the one of H264
+        DataStream videoStream = fac.newVideoOutputH264(getName(), imgWidth, imgHeight);
```

Comment:
Need to set VP8 in output encoding description

Reply:
Done using generic descriptor